test/automation/src/positron/positronQuickInput.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+import { Code } from '../code';
+
+const QUICKINPUT_OK_BUTTON = '.quick-input-widget .quick-input-action a:has-text("OK")';
+
+/*
+ *  Extends Microsoft's QuickInput functionality to provide Positron-specific functionality.
+ */
+export class PositronQuickInput {
+
+	constructor(private code: Code) { }
+
+	async clickOkOnQuickInput(): Promise<void> {
+		await this.code.driver.getLocator(QUICKINPUT_OK_BUTTON).click();
+	}
+}
```

Comment:
I'm fine either way. But if we pull it out, I'd advocate for a separate Positron class that extends the base, to avoid confusion when merging upstream. And then we could always just the new Positron one. (we could also just make that a separate issue/PR)

Reply:
We have been trying not to modify the Microsoft base test automation classes, but perhaps that was not the best choice.  I am not 100% sure why we leaned towards this approach.  I think I originally went with @seeM's opinion on this, but again, I could go either way.  Whatever we do with QuickInput we should also do with our PositronNotebooks.ts class.

Reply:
Interesting, I don't recall having an opinion on this!

If we're only adding one method, I don't see an issue with updating the `QuickInput` class. I actually did that myself when adding the `selectQuickInputElementContaining` method.

Reply:
I must have misremembered.  I will add the function to the existing MS class.